packages/app/universal-ts-utils/package.json

```diff
@@ -39,15 +39,8 @@
         "vitest": "^2.1.3"
     },
     "exports": {
-        "./node": {
-            "types": "./dist/node.d.ts",
-            "import": "./dist/node.js"
-        },
-        "./*": {
-            "types": "./dist/public/*.d.ts",
-            "import": "./dist/public/*",
-            "require": "./dist/public/*"
-        },
+        "./node": "./dist/node.js",
+        "./*": "./dist/public/*.js",
```

Comment:
how does it know where to look for types this way?

Reply:
If types are not specified explicitly, they are picked automatically based on the provided source file path.
Please take a look at https://www.typescriptlang.org/docs/handbook/declaration-files/dts-from-js.html#editing-the-packagejson
![image](https://github.com/user-attachments/assets/eab3cc04-5eea-424d-bc00-ca0a47acf70d)
The same behavior for the `main` field is applied to `exports`.

In general, there is no point to explicitly define `types` unless they are not colocated with the source file.

Reply:
we don't use either "main" or index.d.ts in the root. are you sure this resolution will work?

Reply:
I've tested it in polyglot and it works.
When importing a file with a `.js` extension, the error is reported during `tsc`.
![image](https://github.com/user-attachments/assets/97b3421b-658b-4712-93f7-b002cdae9f06)

After removing the extension, all works as expected (no issues during `tsc`)